doc/tools/nordic_segger.rst

```diff
@@ -154,6 +154,14 @@ To use RTT, you will first need to enable it by adding the following lines in yo
    CONFIG_HAS_SEGGER_RTT=y
    CONFIG_RTT_CONSOLE=y
 
+Additionally you might need to disable other consoles which conflict with the
+RTT one if they are enabled by default in the particular sample or application
+you are running. For example to disable the UART console:
```

Comment:
add:

    For example, to disable the UART console, add this to your ``.conf`` file: